Client/Frontend/Menu/BookmarksViewController.swift

```diff
@@ -188,6 +194,14 @@ class BookmarksViewController: SiteTableViewController {
     
     reloadData()
   }
+    
+    override func viewDidDisappear(_ animated: Bool) {
+        super.viewDidDisappear(animated)
+        // Make sure to remove fetch results controller when view disappears.
+        // Otherwise, it may result in crash if a user is in a nested folder and
+        // sync changes happen.
+        bookmarksFRC = nil
```

Comment:
Curious, why is this causing an actual crash when UI not visible? Is the table deallocated and we're trying to update it or something?

Reply:
Yeah the FRC callbacks strill trigger even thought another view controller was pushed on top of this vc. This causes crash if you move bookmark from root directory to a folder. The `BookmarkVC` of root directory still tries to configure UI.